app/containers/MassEnergizeSuperAdmin/Teams/CreateNewTeamForm.js

```diff
@@ -46,19 +46,21 @@ class CreateNewTeamForm extends Component {
   }
 
   static getDerivedStateFromProps(props, state) {
-    var { communities, formState, location } = props;
+    var { communities, formState, location, auth } = props;
     communities = (communities || []).map((c) => ({
       ...c,
       displayName: c.name,
-      id: "" + c.id,
+      id:c.id,
     }));
+      const isSuperAdmin = auth?.is_super_admin && !auth?.is_community_admin;
```

Comment:
This doesn't look correct.   isSuperAdmin should override isCommunityAdmin.  So if you are both, the admin portal should treat you as superAdmin everywhere.